apps/etterlatte-statistikk/src/main/kotlin/statistikk/config/ApplicationContext.kt

```diff
@@ -43,19 +45,46 @@ class ApplicationContext {
     }
 
     private val behandlingClient: BehandlingClient by lazy {
-        BehandlingClientImpl(httpClient)
+        BehandlingClientImpl(behandlingHttpClient)
     }
 
-    private val statistikkRepository: StatistikkRepository by lazy {
-        StatistikkRepository(datasource)
+    private val statistikkRepository: StoenadRepository by lazy {
+        StoenadRepository(datasource)
     }
 
-    private val sakstatistikkRepository: SakstatistikkRepository by lazy {
-        SakstatistikkRepository(datasource)
+    private val sakstatistikkRepository: SakRepository by lazy {
+        SakRepository(datasource)
     }
     private val datasource: DataSource by lazy { DataSourceBuilder.createDataSource(env).also { it.migrate() } }
 
-    private val httpClient: HttpClient by lazy {
+    private val beregningClient: BeregningClient by lazy {
+        BeregningClientImpl(beregningHttpClient)
+    }
+
+    private val beregningHttpClient: HttpClient by lazy {
+        HttpClient(OkHttp) {
```

Comment:
Dette må jo være en god kandidat for ktor-common sin `httpClientClientCredentials` ?

Reply:
Det gjelder for så vidt behandlingsclienten og

Reply:
TIL!